changelogs/fragments/1083-archive-remove-path-folder.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Maybe someone has been relying on this behaviour and this is a `breaking_change` for them, so I don't know if this should change and wait for the next major release of `community.general` or not.

Reply:
Since the documentation mentions this is supposed to remove trees as well, I think it is OK to do this as a bugfix. If the documentation would not mention that, it would definitely be a breaking change (or would require an option that needs to be turned on explicitly).